### DISCUSSION TITLE
Two images are blended during level-2 processing of Sentinel-2 A/B

### DISCUSSION THREAD

**Discussion by corneliussenf** [davidfrantz/force#71] [Q&A] [answered]
Dear all!

I have processes Sentinel-2 A/B for a study site in northern Italy. While scanning the results, I noticed that some images are mixtures of two input scenes, with a spatial shift between scenes (see screenshot attached). Any idea what could have caused this? Attached is my parameter file for the level-2 processing. I use FORCE v. 3.6.3.

Best

Cornelius

<img width="617" alt="Bildschirmfoto 2021-03-30 um 13 22 13" src="https://user-images.githubusercontent.com/5028407/112983498-10382380-915e-11eb-94a3-26a653dc01e9.png">

[LEVEL2-test2.prm.zip](https://github.com/davidfrantz/force/files/6228740/LEVEL2-test2.prm.zip)

**Comment by davidfrantz** [comment]
Hi Cornelius,

that's plate tectonics for sure!

But honestly, I have never seen anything similar with S2..

The "mixture" is because FORCE merges overlapping images (that are actually the same observation) into one file. This removes the redundancy in the original data.

Did you have a look at the original L1C images? I kind of hope that this alignment issue is not caused by FORCE..
Please also check if the metadata of the L1C product are correct (specifically the UL coordinates, pixel sizes and pixel numbers).

Cheers,
David

**Comment by davidfrantz** [comment, marked as answer]
Another idea:

did you change the datacube settings between processing the images?

**Reply by corneliussenf** [reply]
Oh, that's a good hint! Yes, indeed, I did change the data cube in-between. I thought I deleted all old level-2 images, but maybe some remained. I am going to check the settings and input data and reprocess the data from scratch. Hope the error disappears then. I will let you know (might take me a couple of days though).

Cornelius

P.s. I like the plate-tectonics explanation. Maybe a big earthquake in Italy we didn't know of ;-)

**Reply by corneliussenf** [reply]
I am not 100% sure, but I think changing the data cube settings caused the issue. After rerunning from scratch, everything looks fine.

**Reply by davidfrantz** [reply]
Excellent news! Thanks for reporting back Cornelius!
